java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -1690,16 +1739,6 @@ private void moveDirectory(final String source, final String destination,
         deleteRecursive(source);
     }
 
-    /**
```

Comment:
Removing this will break a lot of libraries that need to do funky things to introspect the state of a `MantaClient`.

Reply:
Hmnn.. Maybe this just got moved. Can you double check?

Reply:
Moved alongside the other getters.